src/react-medium-image-zoom.js

```diff
@@ -138,6 +140,7 @@ export default class ImageZoom extends Component {
       delete this.portalInstance
       delete this.portal
     }
+    this.props.onUnzoom()
```

Comment:
You'll want to pull down the latest code. I would move this to being the second argument to `setState` on [this line of code](https://github.com/rpearce/react-medium-image-zoom/blob/master/src/react-medium-image-zoom.js#L192)